pages/_app.tsx

```diff
@@ -108,6 +108,10 @@ function MyApp<MyAppProps>({ Component, pageProps, userLocation, query }) {
         />
         <meta name="csrf-token" content={csrfToken} />
         {/* <!-- Google Analytics --> */}
+        {/* We can't directly put the script into this component because React
+            doesn't allow it, so we must add it through the
```

Comment:
No, I understand what `dangerouslySetInnerHTML` is; the confusing part was the script itself...

Reply:
Oh, it's the standard script provided by Google to add their analytics in. There's a part for the specific code to be added and that's provided based on if the app is in the QA or production environment.

Reply:
Ohhh.  Ok!